### ISSUE TITLE
High WER Values for a different set

### ISSUE THREAD

**Issue by cemildemir19** [TensorSpeech/TensorFlowASR#206]
We trained a conformer model with Librispeech and our own data set. We used 800 hours of data from Librispeech and 800 hours of our data. In Librispeech test-clean set, we obtained 14.9 % WER. However, for our own test set, we get 63.3 % WER. 
Do you have any other reference result on a different set from Librispeech?
Our data set is too imbalanced. In other words, there are many repeated sentences from different users. For example, we have 1000 different sentences and total number of training samples is about 500.000. That is, every sentence is repeated 500 times in average.

**Comment by usimarit** [comment]
> Do you have any other reference result on a different set from Librispeech?

Unfortunately, I don't have any.

Maybe your own test test is too noisy or has voices that the model didn't see before or the voices are too slow/fast, etc.
Or your training data is too bias, that makes the model 'overfit' on your 800 hours

**Comment by usimarit** [comment]
Feel free to reopen if you have further questions
